aggregator/forms.py

```diff
@@ -47,3 +49,85 @@ def clean_feed_url(self):
                 "here automatically."
             )
         return feed_url
+
+
+class LocalDjangoCommunityModelForm(forms.ModelForm):
+    name = forms.CharField(
+        max_length=250,
+        widget=forms.TextInput(
+            attrs={
+                "class": "required",
+                "placeholder": "Name of the community",
+            }
+        ),
+    )
+    description = forms.CharField(
+        widget=forms.Textarea(
+            attrs={
+                "class": "required",
+                "placeholder": "Description of the community",
+            }
+        ),
+    )
+    continent = forms.CharField(
+        max_length=250,
+        widget=forms.TextInput(
+            attrs={
+                "class": "required",
+                "placeholder": "Continent",
+            }
+        ),
+    )
+    country = forms.ChoiceField(
+        widget=CountrySelectWidget(
+            attrs={
+                "class": "required",
+                "placeholder": "Country",
+            }
+        ),
+        choices=countries,
+    )
+    city = forms.CharField(
+        max_length=250,
+        widget=forms.TextInput(
+            attrs={
+                "class": "required",
+                "placeholder": "City",
+            }
+        ),
+    )
+    website_url = forms.URLField(
+        widget=forms.TextInput(
+            attrs={
+                "placeholder": "Link to the community's website",
+            }
+        ),
+    )
+    event_site_url = forms.URLField(
+        widget=forms.TextInput(
+            attrs={
+                "placeholder": "Link to the community's event site",
+            }
+        ),
+    )
+
+    class Meta:
+        model = LocalDjangoCommunity
+        fields = (
+            "name",
+            "description",
+            "continent",
+            "country",
+            "city",
+            "website_url",
+            "event_site_url",
+        )
```

Comment:
This model contains required `slug` field which should be populated in model before saving this form
You can generate slug with https://docs.djangoproject.com/en/5.0/ref/utils/#django.utils.text.slugify